Assets/Tests/InputSystem/CoreTests_Devices.cs

```diff
@@ -4180,4 +4180,20 @@ public unsafe void Devices_CanSetIMECursorPositionOnKeyboard()
         keyboard.SetIMECursorPosition(Vector2.one);
         Assert.That(commandWasSent, Is.True);
     }
+
+    [Test]
+    [Category("Devices")]
+    public void Devices_RemovingKeyboardMakesNextKeyboardCurrent()
```

Comment:
Ideally should check for inheritance. Maybe `Devices_RemovingDevice_MakesNextDeviceOfTypeCurrent` using a Pointer and a Mouse device.

Reply:
fixed